pkg/cnab/config-adapter/testdata/mybuns-depsv2.bundle.json

```diff
@@ -0,0 +1,207 @@
+      "version": "",
+      "commit": ""
+    },
+    "sh.porter.dependencies.v2": {
+      "requires": {
+        "db": {
+          "name": "db",
+          "bundle": "localhost:5000/mydb:v0.1.0",
+          "parameters": {
```

Comment:
In the cnab spec, we have parameter sources and a well known extension to represent defaulting parameters from outputs of a previous bundle run.

We have two options here. We can add a new version of the parameter source extension to support dependencies, or we can encode the information here. We can't change the existing well known extension because then someone who implemented the original extension would look like they support the changes for dependency parameter sources when they don't.

I suggest that we keep parameter sources separate from "workflow" and "dependencies" so that we can more easily express what's supported by an implementation.